var/spack/repos/builtin/packages/awscli-v2/package.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class AwscliV2(AutotoolsPackage):
```

Comment:
This isn't a real Autotools package, the `configure` script wasn't generated by autoconf. Can you try building with the PythonPackage base class instead?

Reply:
I will try, yes.

Reply:
Actually, I think it is a real Autotools package, but Python is likely better anyway.